packages/core/src/types.ts

```diff
@@ -1735,6 +1746,7 @@ export interface ActorRef<TEvent extends EventObject, TEmitted = any>
   getSnapshot: () => TEmitted | undefined;
   stop?: () => void;
   toJSON?: () => any;
+  parent?: ActorRef<any, any>;
```

Comment:
I would probably only expose this within the actor's implementation - and not on the outside. So people shouldn't be able to do something like that:
```js
sendTo(ctx => ctx.actorRef.parent.parent.parent.parent)
```

Reply:
Right, but that still means that the `parent`/`self` would be available for actors themselves but without the `.parent` property being available on whatever gets access to actor refs.

I've added a TODO comment about this.